beta-template.yml.erb

```
@@ -4,13 +4,13 @@ Description: Provision an instance of the Javabuilder service. Empty the Content
 Parameters:
   BaseDomainName:
     Type: String
-    Description: Base domain name.
+    Description: Base domain name (e.g. 'code.org' in 'javabuilder.code.org').
   BaseDomainNameHostedZonedID:
     Type: String
     Description: AWS Route53 Hosted Zone ID for base domain name.
-  SubDomainName:
+  SubdomainName:
     Type: String
-    Description: Sub domain name for javabuilder service.
+    Description: Sub domain name for javabuilder service (e.g. 'javabuilder' in 'javabuilder.code.org').
```

nit:

```suggestion
    Description: Subdomain name for javabuilder service (e.g. 'javabuilder' in 'javabuilder.code.org').
```

Looks like the corresponding line in `3-app` already got updated!